salt/modules/acme.py

```diff
@@ -95,6 +95,8 @@ def cert(name,
          aliases=None,
          email=None,
          webroot=None,
+         dns_plugin=None,
```

Comment:
These new kwargs need to go at the end of the list so we don't cause an API change and break things for people if they're using positional arguments.

Reply:
Done.